src/App.tsx

```diff
@@ -1,15 +1,30 @@
 import React from 'react';
-
+import { Post } from './types';
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import PostList from './PostList/PostList';
+
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+
+const App: React.FC = () => {
+  const preparedPosts: Array<Post> = posts.map(e => {
+    return {
+      ...e,
+      user: users.find(user => user.id === e.userId) || null,
+      comments: comments.filter(comment => comment.postId === e.id) || null,
+    };
+  });
```

Comment:
fix this and move preparedPosts outside the component

```suggestion
  const preparedPosts: Array<Post> = posts.map(post => ({
      ...post,
      user: users.find(user => user.id === e.userId) || null,
      comments: comments.filter(comment => comment.postId === e.id),
  });
```